tests/e2e/Services/Sites/SitesCustomServerTest.php

```diff
@@ -1275,5 +1275,92 @@ public function testSiteDomainReclaiming(): void
         $this->cleanupSite($site['body']['$id']);
     }
 
+    public function testConsoleAvailabilityEndpoint(): void
+    {
+        $site = $this->createSite([
+            'buildRuntime' => 'ssr-22',
+            'fallbackFile' => null,
+            'framework' => 'other',
+            'name' => 'Test Site',
+            'subdomain' => 'test-site',
+            'outputDirectory' => './',
+            'siteId' => ID::unique()
+        ]);
+
+        $siteId = $site['body']['$id'] ?? '';
+
+        $this->assertEquals(201, $site['headers']['status-code']);
+        $this->assertNotEmpty($site['body']['$id']);
+        $this->assertEquals('Test Site', $site['body']['name']);
```

Comment:
Let's setupSite instead. We should only do createSite, if creation is purpose of the test.